tailwind.config.js

```diff
@@ -20,20 +20,31 @@ module.exports = {
       'secondBlack': '#00204A',
     },
 
+    // width: {
+    //   '16': '3rem',
+    // },
+
     container: {
+      sm: '480px',
+      md: '768px',
+      lg: '1024px',
+      xl: '1440px',
+
       padding: {
         DEFAULT: '1rem',
         sm: '2rem',
         lg: '4rem',
         xl: '5rem',
         '2xl': '6rem',
+        'px-4': '22rem, 19rem',
```

Comment:
why does this need to be there ? wont it override px-4 for all styles?

Reply:
It's a mistake I did it once and cleaned it but came back. Will take it out immediately